### PR TITLE
[8.0] Add web_field_form_color

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Does this conflict with https://github.com/OCA/web/pull/388?

**Comment by ghost** [comment]
@pedrobaeza the functionality is similar indeed. My module is better because the conditions are dynamic.

I keep the same logic as in the list view.

https://github.com/OCA/web/pull/388 uses static intervals / colors and does not use the standard logic of Odoo.

**Comment by pedrobaeza** [comment]
OK, thanks for the info. I have pointed in the other PR to this one then.

File: web_field_form_color/static/src/js/web_field_form_color.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+*/
+
+openerp.web_field_form_color = function (instance) {
+
+    "use strict";
+
+    var QWeb = instance.web.qweb;
+    var _t  = instance.web._t;
+    var _lt = instance.web._lt;
```
Comment (tarteo) [inline web_field_form_color/static/src/js/web_field_form_color.js]: You can shorten this by:

```
var QWeb = instance.web.qweb,
      _t = instance.web._t;
```

variable _lt is never used.

File: web_field_form_color/static/src/js/web_field_form_color.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+    instance.web.form.FieldFloat = instance.web.form.FieldFloat.extend({
```
Comment (tarteo) [inline web_field_form_color/static/src/js/web_field_form_color.js]: Please use .include instead

File: web_field_form_color/static/src/js/web_field_form_color.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+            if(color){
+                this.$el.css('color', color);
+            }
+            else{
+                this.$el.css('color', 'black');
+            }
+        },
+        set_value: function(value_) {
+            this.set_color();
+            return this._super(value_);
```
Comment (tarteo) [inline web_field_form_color/static/src/js/web_field_form_color.js]: Trailing spaces

**Comment by ghost** [comment]
@tarteo thanks for the review. I fixed the code.

**Review by tarteo — APPROVED** [review]
code review 👍

**Comment by joaoalf** [comment]
@pedrobaeza @tarteo @legalsylvain Can we merge this?

**Comment by pedrobaeza** [comment]
Sorry, but I'm not a reviewer of this PR - only a coordinator - and it needs more reviewers.

**Comment by bealdav** [comment]
Not easy to test because, no example of the exact syntax to colors in readme

**Pull request closed without merging**